optimism/src/keccak/lookups.rs

```diff
@@ -24,6 +24,15 @@ pub(crate) trait Lookups {
 
     /// Adds a lookup to the RangeCheck16 table
     fn lookup_rc16(&mut self, rw: LookupMode, flag: Self::Variable, value: Self::Variable);
+
+    /// Adds a lookup to the Reset table
+    fn lookup_reset(
+        &mut self,
+        rw: LookupMode,
```

Comment:
In this environment, we probably only ever want to read. Suggest removing this argument.

Reply:
yes, I am propagating the changes from base branches